service/src/main/java/org/cbioportal/genome_nexus/service/internal/GenomicLocationAnnotationServiceImpl.java

```diff
@@ -88,20 +88,36 @@ public GenomicLocationAnnotationServiceImpl(CachedVariantRegionAnnotationFetcher
     public VariantAnnotation getAnnotation(GenomicLocation genomicLocation)
         throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException
     {
-        return this.variantAnnotationService.getAnnotation(this.notationConverter.genomicToEnsemblRestRegion(genomicLocation));
+        VariantAnnotation variantAnnotation = this.variantAnnotationService.getAnnotation(this.notationConverter.genomicToEnsemblRestRegion(genomicLocation));
+        genomicLocation.setOriginalInput(genomicLocation.toString());
+        variantAnnotation.setOriginalVariantQuery(genomicLocation.getOriginalInput());
+        return variantAnnotation;
     }
 
     @Override
     public VariantAnnotation getAnnotation(String genomicLocation)
         throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException
     {
-        return this.getAnnotation(this.notationConverter.parseGenomicLocation(genomicLocation));
+        VariantAnnotation variantAnnotation = this.getAnnotation(this.notationConverter.parseGenomicLocation(genomicLocation));
+        variantAnnotation.setOriginalVariantQuery(genomicLocation);
+        return variantAnnotation;
     }
 
     @Override
     public List<VariantAnnotation> getAnnotations(List<GenomicLocation> genomicLocations)
     {
-        return this.variantAnnotationService.getAnnotations(this.notationConverter.genomicToEnsemblRestRegion(genomicLocations));
+        Map<String, String> convertedVarsToOrigVarQueryMap = mapConvertedVarsToOrigVarQuery(genomicLocations);
+        List<VariantAnnotation> variantAnnotations = new ArrayList<>();
+        this.variantAnnotationService.getAnnotations(
+                /////////////    WHY IS THE NEXT LINE DIFFERENT LOWER DOWN IN THE CODE? ( this.genomicLocationsToVariantFormats.convert(genomicLocations), )
```

Comment:
was this added here?

which line lower in the code is it referring to? 

maybe a question that @onursumer @leexgh might be able to answer?

Reply:
This comment was left here accidentally. I will remove it, and ask the question with context in a more understandable way.